src/main/java/org/drugis/addis/problems/service/impl/ProblemServiceImpl.java

```diff
@@ -173,13 +173,16 @@ private boolean hasModel(BenefitRiskNMAOutcomeInclusion inclusion) {
           List<BenefitRiskStudyOutcomeInclusion> benefitRiskStudyOutcomeInclusions,
           Map<Integer, Outcome> outcomesById,
           Set<AbstractIntervention> includedInterventions
-  ) {
-    return benefitRiskStudyOutcomeInclusions.stream()
-            .map(inclusion -> getBenefitRiskProblemForInclusion(project, outcomesById, includedInterventions, inclusion))
-            .collect(toList());
+  ) throws IOException {
+    List<BenefitRiskProblem> list = new ArrayList<>();
```

Comment:
we're using streams elsewhere for this type of thing.

I guess you got into this because you don't use try/catch and therefore the exception is leaking out which doesn't play nice with streams. The proper solution imo is to use try/catch with a sensible default in case the title retrieval fails.